project/templates/app/mobi/app/includes/comments_include_no_likes.html

```diff
@@ -28,10 +28,10 @@
     </span>
 
     {% if comment.is_removed and comment.user.profile.banned %}
-    <p>{% trans "This comment has been reported by the community and the user has been banned" %}<br><br/>
+    <p>{% trans "This comment has been removed by a moderator and the user has been banned from commenting for 3 days." %}<br><br/>
```

Comment:
This was removed by the community so it should be `Your comment has been flagged by a community member. You will not be able to comment again until tomorrow.`